realsense/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(RealSenseImShowExample)
+
+# OpenCV
+find_package(OpenCV REQUIRED)
+include_directories(${OpenCV_INCLUDE_DIRS})
+
+# Intel RealSense
+find_package(realsense2 REQUIRED)
+include_directories("${realsense2_INCLUDE_DIR}")
+
+add_definitions(-Wno-format-extra-args -std=c++17 -g -O3)
```

Comment:
Add `-Wall -Wextra -Werror`. The compiler will complain more, but it's usually worth it. Message me the new warnings this introduces, if any.